sale_crm_medical_prescription/wizards/medical_lead_wizard.py

```diff
@@ -75,7 +75,7 @@ def action_create_leads(self, ):
                 'pharmacy_id': self.pharmacy_id.id,
                 'prescription_order_line_ids': order_lines,
                 'is_prescription': True,
-                'name': ', '.join(l.name for l in order)
+                'name': ', '.join([l.name or '' for l in order])
             })
 
         _logger.debug('Created %s', lead_ids)
```

Comment:
Let's remove this line, it's lowering our test coverage & doesn't really seem needed in long term